src/DevilDaggersInfo.Web.Server.Domain/Models/ModArchives/ModBinaryCacheData.cs

```diff
@@ -8,19 +8,19 @@ namespace DevilDaggersInfo.Web.Server.Domain.Models.ModArchives;
 
 public class ModBinaryCacheData
 {
-	public ModBinaryCacheData(string name, long size, ModBinaryType modBinaryType, List<ModChunkCacheData> chunks, List<ModifiedLoudnessAssetCacheData>? modifiedLoudnessAssets)
+	public ModBinaryCacheData(string name, long size, ModBinaryType modBinaryType, List<ModTocEntryCacheData> tocEntries, List<ModifiedLoudnessAssetCacheData>? modifiedLoudnessAssets)
 	{
 		Name = name;
 		Size = size;
 		ModBinaryType = modBinaryType;
-		Chunks = chunks;
+		TocEntries = tocEntries;
 		ModifiedLoudnessAssets = modifiedLoudnessAssets;
 	}
 
 	public string Name { get; }
 	public long Size { get; }
 	public ModBinaryType ModBinaryType { get; }
-	public List<ModChunkCacheData> Chunks { get; }
+	public List<ModTocEntryCacheData> TocEntries { get; }
```

Comment:
Add JSON property name workaround + unit test to verify JSON deserialization of existing files.